### ISSUE TITLE
Outputting a pom.xml from dependencies.yml

### ISSUE THREAD

**Issue by Fraserhardy** [playframework/play1#1336]
We'd like to improve the security of our application by scanning dependencies. The majority of tooling is only able to scan a Pom.xml file for dependencies to build up a BOM. 

I have looked into this and it seems this would need changes to the play dependencies module. Does anyone know if this is possible directly within ivy settings without changes to Play? 

Also it would be helpful to have dependency checking of Play framework itself as at the moment it's difficult to know if Play is using vulnerable dependencies.

**Comment by cies** [comment]
I do know that RePlay (a fork of Play1) gives you "just a Gradle project" (no funky/clunky Python, and old Ivy2 stuff).

**Comment by Fraserhardy** [comment]
Yeah I like what the team on RePlay did. I'm not sure how easy it is to port an existing app to RePlay though?
